### ISSUE TITLE
Build example doesn't work

### ISSUE THREAD

**Issue by Penguinz22** [gabime/spdlog#1974]
If I try and build the example with the following commands on Windows 10:
`cd example`
`cmake -G "MinGW Makefiles" ..`
`make`

When I run the `make` command it takes about 20 to 30 seconds to compile which I feel to too long where it's not normal and when I go and run the example.exe nothing outputs to console. Am I doing something wrong here?

**Comment by tt4g** [comment]
Is the example project really built?

Normally, the `make` command should not exist in Windows.
And if you have changed the current directory to `example` with `cd example`, then `cmake -G "MinGW Makefiles" ..` must be `cmake -G "MinGW Makefiles" .` (`CMakeLists.txt` for example project is `example/CMakeLists.txt`).

**Comment by tt4g** [comment]
If `example.exe` has been built, check the return code of `example.exe`.

Example:
```bash
$ example.exe
$ echo %ERRORLEVEL%
```

Return codes other than `0` will be displayed in case `example.exe` is not executed due to an error such as an ABI-incompatible binary was generated or the library required for execution was not loaded.

**Comment by Penguinz22** [comment]
If I try and run it from command prompt I get this
![image](https://user-images.githubusercontent.com/22716008/123003349-ef035f80-d367-11eb-8ff4-838ea11c7b6a.png)

**Comment by tt4g** [comment]
A binary has been generated that cannot be executed by the Windows OS.
You should not build it with `make` on your machine.

Read the CMake documentation (https://cmake.org/cmake/help/latest/guide/tutorial/index.html) and run the build command that fits your environment.

Example:

```bash
$ cd example
$ cmake .
$ cmake --build .
```

**Comment by Penguinz22** [comment]
I reran the cmake command this time building in the correct directory though I'm getting cmake errors now which I honestly don't quite understand. Here is the command line and attached, the cmake log.
```PS C:\Users\Nathan\Desktop\spdlog-1.x\example> cmake -G "MinGW Makefiles" .
-- The CXX compiler identification is GNU 8.1.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/mingw-w64/x86_64-8.1.0-posix-seh-rt_v6-rev0/mingw64/bin/g++.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:8 (find_package):
  By not providing "Findspdlog.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "spdlog", but
  CMake did not find one.

  Could not find a package configuration file provided by "spdlog" with any
  of the following names:

    spdlogConfig.cmake
    spdlog-config.cmake

  Add the installation prefix of "spdlog" to CMAKE_PREFIX_PATH or set
  "spdlog_DIR" to a directory containing one of the above files.  If "spdlog"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
See also "C:/Users/Nathan/Desktop/spdlog-1.x/example/CMakeFiles/CMakeOutput.log".
[CMakeOutput.log](https://github.com/gabime/spdlog/files/6697754/CMakeOutput.log)

**Comment by tt4g** [comment]
It appears that you do not have spdlog installed.
Without spdlog installed, the build of the example project will fail.

**Comment by Penguinz22** [comment]
Ah, so I was trying to build the example project due to spdlog not working properly in my main project. I finally build the example project properly. And with the dll error I fixed it by finding it was not using the dll by the same name from the MinGW compiler but from QEMU that I previous installed on my machine, once I removed that dll from QEMU it started using the correct one and the example ran with no errors.
